src/main/java/com/glodblock/github/client/gui/GuiMagnet.java

```diff
@@ -0,0 +1,178 @@
+package com.glodblock.github.client.gui;
+
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.entity.player.InventoryPlayer;
+import net.minecraft.util.ResourceLocation;
+
+import com.glodblock.github.FluidCraft;
+import com.glodblock.github.client.gui.container.ContainerMagnet;
+import com.glodblock.github.client.gui.widget.FCGuiBaseButton;
+import com.glodblock.github.client.gui.widget.GuiFCImgButton;
+import com.glodblock.github.common.item.ItemMagnetCard;
+import com.glodblock.github.common.item.ItemWirelessUltraTerminal;
+import com.glodblock.github.inventory.InventoryHandler;
+import com.glodblock.github.network.CPacketFluidTerminalBtns;
+import com.glodblock.github.util.NameConst;
+
+import appeng.api.config.ActionItems;
+import appeng.api.config.Settings;
+import appeng.api.storage.ITerminalHost;
+import appeng.client.gui.AEBaseMEGui;
+import appeng.client.gui.widgets.GuiImgButton;
+import appeng.client.gui.widgets.GuiTabButton;
+
+public class GuiMagnet extends AEBaseMEGui {
+
+    protected FCGuiBaseButton listModeBtn;
+    protected ContainerMagnet cont;
+    private static final ResourceLocation TEX_BG = FluidCraft.resource("textures/gui/magnet_filter.png");
+
+    protected Component components[] = new Component[3];
+    protected GuiTabButton originalGuiBtn;
+    protected GuiImgButton clearBtn;
+
+    public GuiMagnet(InventoryPlayer ip, ITerminalHost container) {
+        super(new ContainerMagnet(ip, container));
+        this.xSize = 195;
+        this.ySize = 214;
+        this.cont = (ContainerMagnet) this.inventorySlots;
```

Comment:
in is better to use full name for variables and if using this exact type is necessary instanceof should be added